contributor_ladder.md

```diff
@@ -0,0 +1,188 @@
+# Konveyor Contributor Ladder
+
+* [Contributor Ladder](##contributor-ladder-template)
+    * [Community Participant](###contributor-participant)
+    * [Project Contributor](###project-contributor)
+    * [Project Member](##project-member)
+    * [Project Reviewer](###project-reviewer)
+    * [Project Maintainer](###project-maintainer)
+* [Inactivity](##inactivity)
+* [Involuntary Removal](##involuntary-removal)
+* [Stepping Down/Emeritus Process](##stepping-downemeritus-process)
+* [Contact](##contact)
+
+
+## Contributor Ladder
+
+Hello! We are excited that you want to learn more about contributing to Konveyor. This "contributor ladder" outlines the different contributor roles within the project, along with the responsibilites and privileges that come with them. Community members generally start at the first levels of the ladder and advance up it as their involvement in the project grows.  Our project members are happy to help you advance along it.
+
+Each of the contributor roles below is organized into lists of three types of things. "Responsibilities" are things that contributor is expected to do. "Requirements" are qualifications a person needs to meet to be in that role, and "Rights" are things contributors on that level are entitled to.
+
+## Individual Project Roles
+
+Since the Konveyor umbrella project is made up of multiple member projects, your contributor role is *per-project*. After Community Participant, contributors progress along the ladder in each project separately.  This means that you can easily be a Maintainer on one project, while being a Contributor in another, and a Reviewer in a third.
+
+Collectively, contributors are also contributors at their highest level to the Konveyor Umbrella project.  That is, if you are a Project Member of Forklift, you are considered a Member of the whole Konveyor project, particularly for governance purposes.
+
+### Community Participant
+
+Description: A Community Participant participates in the community and contributes their time, thoughts, etc. Community participants are usually users who have stopped being anonymous and started being active in project discussions.
+
+* Responsibilities:
+    * Must follow the Konveyor CoC
+* How users can get involved with the community:
+        * Participating in community discussions
```

Comment:
Could/should we include references/links to where "community discussions" happen? I assume we mean the google groups, but even I'm not sure.

Reply:
I'd rather not do that right here?  That would give us *three* places to maintain that information over time, since the list of community channels is already in the README and on the website.  I really don't want to add a third place.

Reply:
Linked to README instead.